plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/wizard/IKeyValueItem.java

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.common.ui.wizard;
+
+public interface IKeyValueItem {
```

Comment:
@adietish I was preferring to not alter the express code as much as possible, but I would say this is more generic then env vars.  This was introduces to handle labels and env in a common way similar to what appears in the web console.  This should provide a common mechanism to do key/value pairs with custom validations, etc.  EnvironmentVariablesItem is a variant of this.